src/utils/pathwayAnalysis/pathwayServices.js

```diff
@@ -0,0 +1,21 @@
+import {
+  H_SAPIENS, M_MUSCULUS, D_MELANOGASTER, S_CEREVISAE, C_ELEGANS, D_RERIO,
+} from 'utils/pathwayAnalysis/pathwaySpecies';
+
+const pathwayServices = { PANTHERDB: 'pantherdb', ENRICHR: 'enrichr' };
+
+const serviceUrls = {
+  [pathwayServices.ENRICHR]: {
+    [H_SAPIENS.value]: 'https://maayanlab.cloud/Enrichr/enrich',
+    [M_MUSCULUS.value]: 'https://maayanlab.cloud/Enrichr/enrich',
+    [D_MELANOGASTER.value]: 'https://maayanlab.cloud/FlyEnrichr/enrich',
+    [S_CEREVISAE.value]: 'https://maayanlab.cloud/YeastEnrichr/enrich',
+    [C_ELEGANS.value]: 'https://maayanlab.cloud/WormEnrichr/enrich',
+    [D_RERIO.value]: 'https://maayanlab.cloud/FishEnrichr/enrich',
+  },
+};
+
+export default pathwayServices;
+export {
+  serviceUrls,
+};
```

Comment:
This file is storing many things, why not rename it to `pathwayAnalysisConstants.js`, and move `pathwaySpecies` into this? And just export everything with now default maybe (even though eslint gets angry).

Also `pathwaySpecies` is only `enrichrSpecies`.

Reply:
So the species dropdown isn't used for PantherDB?